### DISCUSSION TITLE
Caching of Coverity

### DISCUSSION THREAD

**Discussion by troglobit** [lldpd/lldpd#460] [General]
Hi Vincent,

just wanted to let you know, I copied your coverity.yml workflow to several of my projects due to the migration from Travis-CI.  I've added caching and run-on a dedicated branch, among other things.  I think at least the caching part could perhaps be useful to feedback to you.

See here https://github.com/troglobit/smcroute/actions/runs/1070942775/workflow

Cheers
 /Joachim

**Comment by vincentbernat** [comment]
Thanks! I have borrowed your code. Let's see if I didn't make a copy/paste mistake!

**Reply by troglobit** [reply]
Good luck! :sunglasses::+1:
